clang/test/CheckedCRewriter/extGVarbar1.c

```diff
@@ -1,5 +1,6 @@
 //RUN: cconv-standalone -base-dir=%S -output-postfix=checked %s %S/extGVarbar2.c
 //RUN: FileCheck -match-full-lines --input-file %S/extGVarbar1.checked.c %s
+//RUN: %clang -c %S/extGVarbar1.checked.c %S/extGVarbar2.checked.c
```

Comment:
What is the source of failure here? (And should it be a BUG test if it's failing?)

Reply:
(Same question for other tests below with `clang` call commented out.)

Reply:
oh n/m. This is just referring to generic clang, I guess? I was interpreting `%clang` as the call being commented out.